desktop/apps/auctions2/reducers/appReducer.js

```diff
@@ -1,17 +1,39 @@
 import * as types from 'desktop/apps/auctions2/actions/appActions'
 import u from 'updeep'
+import { filter, reject } from 'underscore'
 
 export const initialState = {
-  isMobile: false
+  isMobile: false,
+  auctions: {
+    live: [],
+    timed: []
+  },
+  isFetchingAuctions: false
 }
 
 export default function appReducer (state = initialState, action = {}) {
   switch (action.type) {
-    case types.TEST_ACTION: {
-      console.log('Hello test action!')
-
+    case types.GET_AUCTIONS_FAILURE: {
+      return u({
+        isFetchingAuctions: false
+      }, state)
+    }
+    case types.GET_AUCTIONS_REQUEST: {
+      return u({
+        isFetchingAuctions: true
+      }, state)
+    }
```

Comment:
This is minor, but i would move `GET_AUCTIONS_REQUEST` just above `GET_AUCTIONS_FAILURE` so that the code reads similar to how things happen IRL: "Make request: -> Failure? -> Success?"